app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/JumpToButton.tsx

```diff
@@ -8,17 +8,17 @@ import i18n from "../../utils/i18n";
 
 export interface JumpToButtonProps {
     blocks: BlockDef[];
+    expanded: boolean;
 }
 
-export const JumpToButton: React.FC<JumpToButtonProps> = React.memo(props => {
-    const { blocks } = props;
+export const JumpToButton: React.FC<JumpToButtonProps> = React.memo(({ blocks, expanded }) => {
     const items: DropdownProps["items"] = React.useMemo(() => {
         return blocks.map(block => ({ id: block.id, text: block.title }));
     }, [blocks]);
 
     return (
         <Dropdown
-            text={i18n.t("Jump to")}
+            text={(expanded && i18n.t("Jump to")) || undefined}
```

Comment:
[best practices]  Typically `a && b || c` (old python style) can be simply written `a ? b : c`